cmd/flat-network-controller/flat-network-controller.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The external network controller is responsible for running controller loops for the flat network providers.
+// Most of canonical CNI plugins can be used on so-called flat networks.
+
+package main
+
+import (
+	"flag"
+	arktosext "k8s.io/arktos-ext/pkg/generated/clientset/versioned"
+	"k8s.io/arktos-ext/pkg/generated/informers/externalversions"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/cmd/flat-network-controller/app"
+	"time"
+)
+
+const defaultWorks = 4
```

Comment:
Nit: name of the variable, work or worker?

Reply:
worker. will update.